### ISSUE TITLE
Wrong ruby dependecy

### ISSUE THREAD

**Issue by cyupa** [CocoaPods/CocoaPods#4099]
I've installed **cocoapods 0.38.2** and somehow ended up with **cocoapods-trunk version 0.6.3** instead of version 0.6.1.
Because cocoapods 0.38.2 is dependent to the **nap 0.8** ruby gem while cocoapods-trunk 0.6.3 is depended to **nap 1.0** that generated some error when running any pod command:
```
pod install
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:2007:in `raise_if_conflicts': Unable to activate cocoapods-trunk-0.6.3, because nap-0.8.0 conflicts with nap (~> 1.0) (Gem::LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1176:in `activate'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1212:in `block in activate_dependencies'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1198:in `each'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1198:in `activate_dependencies'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/specification.rb:1180:in `activate'
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_gem.rb:48:in `gem'
	from /usr/bin/pod:22:in `<main>'
```

I installed cooapods on a fresh installed OS X 10.10.5.

I was able to **fix** this by uninstalling version 0.6.3 of cocoapods-trunk:
`sudo gem uninstall cocoapods-trunk -v 0.6.3`
And then installing the version 0.6.1
`sudo gem install cocoapods-trunk -v 0.6.1`

**Comment by segiddins** [comment]
Duplicate of #4095

**Comment by segiddins** [comment]
This should now be fixed with the release of `cocoapods-trunk (0.6.4)`.
